services/name/src/test/java/org.aws.samples.compute.name/NamesTest.java

```diff
@@ -0,0 +1,69 @@
+package org.aws.samples.compute.name;
```

Comment:
Directory name does not look correct? It should be `org/aws/samples/compute/name`.